servicecontrol/troubleshooting.md

```diff
@@ -328,3 +328,22 @@ HTTP Error 400. The request hostname is invalid.
 
 The hostname can be configured via the [config file 'ServiceControl/HostName' ](/servicecontrol/creating-config-file.md#host-settings-servicecontrolhostname) or the [SCMU hostname](/servicecontrol/setting-custom-hostname.md) setting.
 
+## VoronUnrecoverableErrorException
+
+This is an indication that something has gone very wrong with the RavenDB database. It is recommended to contact [Particular support](https://particular.net/support) for assistance. To recover the database, perform the following:
+
+Use the `Tools` download link on the [RavenDB](https://ravendb.net/download) site to download the `Voron.recover.exe` tool. Once downloaded, extract the archive and open an admin console. Navigate to the tool folder, and run:
+
+.\Voron.Recovery.exe recover "[DBFOLDERPATH]\Databases\audit" "C:\DBRecoverFolder"
```

Comment:
By indenting it with atleast 4 spaces it will use code formatting
```suggestion
    .\Voron.Recovery.exe recover "[DBFOLDERPATH]\Databases\audit" "C:\DBRecoverFolder"
```